src/config/constants.ts

```diff
@@ -29,6 +29,12 @@ export const DEFAULT_BLOCKAID_SCAN_DELAY = 1000;
 // This is used to prevent flickering while refreshing lists with "pull to refresh" action
 export const DEFAULT_REFRESH_DELAY = 1000;
 
+// Balances polling interval in milliseconds
+export const BALANCES_FETCH_POLLING_INTERVAL = 60000;
```

Comment:
since we are removing a redundant bulk-scan request on the frontend and only fetching on focus I think it'd be worth to keep it as 30secs so users don't need to wait too long to see a balance update